internal/storage/download_test.go

```diff
@@ -56,7 +63,47 @@ func TestDownloadToFile(t *testing.T) {
 			HashType:  "SHA256",
 			HashValue: "4eefb9a7a40a8b314b586a00f307157043c0bbe4f59fa39cba88773680758bc3",
 		},
+	}, t)
+}
+
+// TestDownloadToFileSecure tests downloadToFile function, using secure protocol(s).
+func TestDownloadToFileSecure(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		t.Skip("this test only runs on Linux.")
 	}
+	sslCertFile := os.Getenv("SSL_CERT_FILE")
+	certFiles := strings.Split(sslCertFile, string(os.PathListSeparator))
```

Comment:
Duplicate.

Reply:
The purpose of certFiles array is to contain the split value of SSL_CERT_FILE environmental variable. It can contain one or multiple paths, separated by  os.PathListSeparator. After splitting, the array is passed to the for loop on the next line, where every element is tested if ends with the CA filename.

Reply:
Actually, the purpose is just to validate that the test certificate is manually added.